File: mobile/android/app/src/main/java/ru/edu/asu/minobrlabs/db/entities/Annotation.java
package ru.edu.asu.minobrlabs.db.entities;

import java.io.Serializable;

public class Annotation implements Serializable {
    public Long _id;

    public Long experiment;
    public Integer param;
    public Long time;
    public String text;

    public Annotation() {}

    public Annotation(final Long experiment, final Integer param, final Long time, final String text) {
        this.experiment = experiment;
        this.param = param;
        this.time = time;
        this.text = text;
    }

    @Override
    public String toString() {
        return String.format(
                "Annotation : [\n  _id : %s\n  experiment : %s\n  param : %s\n  time : %s\n  text : %s\n]",
                _id, experiment, param, time, text
        );
    }
}

File: desktop/windows/MinobrLabsProject/MinobrLabsProject/db/entities/Annotation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MinobrLabsProject.db.entities
{
    public class Annotation
    {
        public long experiment;
        public int param;
        public long time;
        public string text;

        public Annotation() { }

        public Annotation(long experiment, int param, long time, string text)
        {
            this.experiment = experiment;
            this.param = param;
            this.time = time;
            this.text = text;
        }
        
        public string toString()
        {
            return string.Format(
                    "Annotation : [\n  experiment : %s\n  param : %s\n  time : %s\n  text : %s\n]",
                    experiment, param, time, text
            );
        }
    }
}
